src/Controller/AbsenceBlockController.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Controller;
+
+use App\Controller\BaseController;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\Routing\Annotation\Route;
+
+use App\Model\AbsenceBlock;
+
+class AbsenceBlockController extends BaseController
+{
+    /**
+     * @Route("/absence/block", name="absence.block.index", methods={"GET"})
+     */
+    public function index(Request $request, Session $session)
+    {
+        $today = date('Y-m-d');
+
+        $queryBuilder = $this->entityManager->createQueryBuilder();
+
+        $query = $queryBuilder->select(array('a'))
+            ->from(AbsenceBlock::class, 'a')
+            ->orderBy('a.start', 'ASC', 'a.end', 'ASC')
+            ->getQuery();
```

Comment:
On doit pouvoir simplifier ceci (tant qu'on n'utilise pas de filtre sur les dates) par 
`$blocks = $this->entityManager->getRepository(AbsenceBlock::class)`
`    ->findBy(`
`        array(),`
`        array('start' => 'ASC', 'end' => 'ASC'),`
`    );`

Reply:
Le filtre sur les dates me semble intéressant.
Dans ce cas, on conserverait la variable $today et le QueryBuilder pour ajouter le filtre `a.end >= $today`

Reply:
fixed by a913a36e4144640a9a040cb71f9acd9994bc1ccf